README.md

```diff
@@ -194,7 +194,7 @@ provisioner: kubernetes.io/portworx-volume
 parameters:
    repl: "2"
 ---
-apiVersion: apps/v1beta1
+apiVersion: apps/v1
```

Comment:
You'll also need the selector under spec.

Reply:
Added selector for labels.

Reply:
The selector you added is in the wrong place.  If I try to apply this yaml I get:

```
persistentvolumeclaim/mysql-data created
storageclass.storage.k8s.io/px-mysql-sc created
error: error validating "a.yml": error validating data: [ValidationError(Deployment.spec.template.spec): unknown field "selector" in io.k8s.api.core.v1.PodSpec, ValidationError(Deployment.spec): missing required field "selector" in io.k8s.api.apps.v1.DeploymentSpec]; if you choose to ignore these errors, turn validation off with --validate=false
```

Reply:
You need to do something like this:

```
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
   name: mysql-data
   annotations:
     volume.beta.kubernetes.io/storage-class: px-mysql-sc
spec:
   accessModes:
     - ReadWriteOnce
   resources:
     requests:
       storage: 2Gi
---
kind: StorageClass
apiVersion: storage.k8s.io/v1
metadata:
    name: px-mysql-sc
provisioner: kubernetes.io/portworx-volume
parameters:
   repl: "2"
---
apiVersion: apps/v1
kind: Deployment
metadata:
  name: mysql
spec:
  strategy:
    rollingUpdate:
      maxSurge: 1
      maxUnavailable: 1
    type: RollingUpdate
  replicas: 1
  selector:
    matchLabels:
      app: mysql  # needs to match spec.template.metadata.labels
  template:
    metadata:
      labels:
        app: mysql
        version: "1"
    spec:
      schedulerName: stork
      containers:
      - image: mysql:5.6
        name: mysql
        env:
        - name: MYSQL_ROOT_PASSWORD
          value: password
        ports:
        - containerPort: 3306
        volumeMounts:
        - name: mysql-persistent-storage
          mountPath: /var/lib/mysql
      volumes:
      - name: mysql-persistent-storage
        persistentVolumeClaim:
          claimName: mysql-data
```

Reply:
See this example: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components

The indentation is wrong in this example, but the comments give you an idea of what you need to do when adding a spec.selector

Reply:
This is the same example, but with the indentation fixed:
https://github.com/kubernetes/website/blob/master/content/en/docs/concepts/workloads/controllers/statefulset.md#components

Reply:
This has been fixed.

Reply:
With the latest changes, this file now works with `kubectl apply`